nodejs/eks/yarn.lock

```diff
@@ -35,7 +35,7 @@
   resolved "https://registry.yarnpkg.com/@logdna/tail-file/-/tail-file-2.2.0.tgz#158a362d293f940dacfd07c835bf3ae2f9e0455a"
```

Comment:
Curious - why doesn’t this lock file end up being used?  Is it because we use yarn here and end users use npm?  Cc @Frassle

Reply:
The lock file is lost somewhere during the publishing pipeline. If we don't get vercel/pkg implemented soon, we should instead investigate why this is excluded.

Reply:
The secondary issue with the lock file is that we've only got a yarn.lock, but if the user doesn't have yarn available then it won't help.

Reply:
> end users use npm

The engine just does an "npm install" when installing dependencies.

Reply:
Should do

Reply:
See https://github.com/pulumi/pulumi/issues/11147 - we should only ever use `npm ci` or `yarn install --frozen-lockfile`